release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1537,6 +1537,14 @@ conditions:
 * Previously, when creating an update service application through the web console, an invalid host error occurred. This occurred because the default OpenShift Update Service (OSUS) application name was too long. A shorter default name is now in place and the error no longer occurs.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939788[*BZ#1939788*])
 
+[discrete]
+[id="ocp-4-9-performance-addon-operator-bug-fixes"]
+==== Performance Addon Operator
+
+The following update for the Performance Addon Operator is now available for {product-title} 4.9:
+
+* Previously, the Performance Addon Operator could not restart properly in environments where the connection is bandwidth limited. It also could not restart properly when single node clusters, or any other edge nodes, lost connection to the image registry. With this update, the issue is resolved by ensuring the image is not pulled from `registry.redhat.io` if the image is already available on the node. This fix ensures that the Performance Addon Operator restarts correctly using the image from the local image cache. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055019[*BZ#2055019*])
```

Comment:
Is _where the connection is bandwidth limited_ the best way to state this? 
_with a bandwidth-limited connection_ ??